src/System.Runtime/tests/System/TimeZoneInfo.cs

```diff
@@ -10,6 +10,8 @@
 public static class TimeZoneInfoTests
 {
     private static readonly bool s_isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+    private static readonly bool s_isOSX = RuntimeInformation.IsOSPlatform(OSPlatform.OSX);
```

Comment:
I am ok with your change but here is another idea you may consider it.

[Theory]
[InlineData(... whatever Linux Specific data...)]
[PlatformSpecific(PlatformID.Linux)]
public static void LinuxNearMinMaxDateTimeOffsetConvertTest(..Linux data...)
{
    NearMinMaxDateTimeOffsetConvertTest(data);
}

[Theory]
[InlineData(... whatever Windows Specific data...)]
[PlatformSpecific(PlatformID.Windows)]
public static void WindowsNearMinMaxDateTimeOffsetConvertTest(..Windows data.)
{
    NearMinMaxDateTimeOffsetConvertTest(data);
}

public static void NearMinMaxDateTimeOffsetConvertTest(data)
{
 ...
}